### PR TITLE
[RLlib] Better behavior if user does not specify stopping condition in RLLib CLI

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
Signed-off-by: Artur Niederfahrenhorst <artur@anyscale.com>

## Why are these changes needed?

We have lots of examples inside RLlib of running ray CLI similar to `rllib train --env ... --config ...` that lack a stopping condition.
They error out with the following message.

Screenshot 2022-12-13 at 23 06 18

We should not error out, since a workflow to be expected from us is to quickly start an experiment from CLI, observe it and cancel it manually without caring for stopping condidations.

## Related issue number

https://discuss.ray.io/t/problem-with-rllib-configuration-by-command-line/8624

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/common.py [hunk 1]
```diff
@@ -216,7 +218,7 @@ class CLIArguments:
 
     # Train arguments
     # __cli_train_start__
-    Stop = typer.Option(None, "--stop", "-s", help=get_help("stop"))
+    Stop = typer.Option(None, *STOP_ARG_NAMES, help=get_help("stop"))
```
Comment (maxpumperla) [inline rllib/common.py:221]: @ArturNiederfahrenhorst hmm, I think you can fix all of this by making the default arg `"{}"` instead of `None`

**Review by maxpumperla — CHANGES_REQUESTED** [review]
great, thanks for spotting this @ArturNiederfahrenhorst! I think we can simplify

**Comment by ArturNiederfahrenhorst** [comment]
@maxpumperla I have also removed the warning I introduced about stop being empty to simplify this.
Do you think that's a good idea or would you like it back?

**Comment by maxpumperla** [comment]
@ArturNiederfahrenhorst I think we can add this to the typer help of the stop command, if not already there

**Review by maxpumperla — APPROVED** [review]

**Comment by ArturNiederfahrenhorst** [comment]
@maxpumperla Info on how to use stop is already in the help string. I'll leave it there and not warn users when it's missing.

**Pull request merged**